scripts/generate-sdk/generate-sdk.sh

```diff
@@ -78,14 +78,14 @@ go)
 
     source ${LANGUAGE_GENERATORS_FOLDER_PATH}/${LANGUAGE}.sh
     # Usage: generate_go_sdk GENERATOR_PATH GIT_HOST GIT_USER_ID [GIT_REPO_ID] [SDK_REPO_URL]
-    generate_go_sdk ${jar_path} ${GIT_HOST} ${GIT_USER_ID} ${GIT_REPO_ID} ${SDK_REPO_URL}
+    generate_go_sdk "${jar_path}" "${GIT_HOST}" "${GIT_USER_ID}" "${GIT_REPO_ID}" "${SDK_REPO_URL}"
```

Comment:
Any reason why we need to change this now? It would be more resilient to strings with empty spaces, but I don't think any of these variables should have an empty space

Reply:
This PR is still in draft! I did this temporarily to be able to test my changes locally. As discussed yesterday I've already moved these changes to a different branch so the actual changes to the template are the only changes for this PR